racketlogger/ModernKiosk
# Improvements with EKC, password, and, notifications

Also this commit add:
* Aligment with Firefox Photon Design and Icons
* Fix some bugs in menus
* Add icons
* Licences
* JSDoc

To test:
- Go to preferences change preferences, password, etc.
- Test Modern Kiosk Mode (fullscreen)
- Test menu in context menu (mouse)
- Disable context menu in preferences
- Test EKC (Ctrl + Shift + U)
- Test to disable password
- Test to change password.

PS.:
- Please see copyright for RacketLogger if not fit, please change in a new commit.
- This will be a legacy version, since firefox --url <url> --kiosk in version 71.0a1 Firefox Nightly
